### ISSUE TITLE
SSL required in docker-compose.insecure.yml

### ISSUE THREAD

**Issue by Amiiya** [SUSE/Portus#1449]
Hello!

Relating to this issue: https://github.com/SUSE/Portus/issues/1430 .
I ensured that the environment variable `PORTUS_CHECK_SSL_USAGE_ENABLED` in docker-compose.insecure.yml is set to `false`.
But when setting it up it still shows the same error.
```
Please review your configuration
SSL

For security reasons you need to use SSL. SSL is not configured, please review your server configuration.
```
Is this a bug or am I supposed to change the variable somewhere else too?

Kind regards,
Amiya x

**Comment by mssola** [comment]
Hi @Amiiya, thanks for reporting. I'm sure this is a duplicate of #1418. For now you can use `mssola/portus:susecon` until we get it fixed (which should happen rather soonish)

**Comment by bosr** [comment]
Thank you @mssola !

**Comment by roboto-cloud** [comment]
@mssola 
this two issues was resolved whit your image `mssola/portus:susecon`
https://github.com/SUSE/Portus/issues/1507
https://github.com/SUSE/Portus/issues/1508

Thanks!

**Comment by mssola** [comment]
This should probably be fixed with #1526. Until that is merged, you can probably use the `CCONFIG_PREFIX=PORTUS` environment variable option.

**Comment by amccool** [comment]
`CCONFIG_PREFIX=PORTUS` did not work for me.
however
`mssola/portus:susecon` got the insecure example running (at least the UI).

**Comment by mssola** [comment]
Then it's definitely something on the package. Let's see if we can get #1418 fixed asap...

**Comment by mssola** [comment]
Fixed! See my comments on #1418.
